openapi/openapi_utils.go

```diff
@@ -330,7 +335,7 @@ func LoadOASFromFileOrNetwork(log *logrus.Logger, env config.EnvironmentVariable
 		return oas, nil
 	}
 
-	return nil, fmt.Errorf("missing environment variables one of %s or %s is required", config.TargetServiceOASPathEnvKey, config.APIPermissionsFilePathEnvKey)
+	return nil, fmt.Errorf("missing openapi config: one of TargetServiceOASPath or APIPermissionsFilePath is required")
```

Comment:
The problem with this change in the error is that it will be logged with an incomplete information for the user; it will see the name of the field instead of the env var that should be set.

The error in itself is fine, but I'd wrap it and make a check in the main function to provide further information to the user about the env var that are missing.

Reply:
Added in the config package